### PR TITLE
feat(sibling-box): add element customization

### PR DESCRIPTION
**Pull request by SiTaggart**
Add element customization to Sibling Box for customizaing checkbox and radio button

### CODE REVIEW

File: packages/paste-core/primitives/sibling-box/src/index.tsx [hunk 1]
```diff
@@ -77,21 +79,34 @@ const SiblingBox = styled(Box)<SiblingBoxProps>(
...
 
+const SiblingBox: React.FC<SiblingBoxProps> = ({
+  'aria-hidden': ariaHidden = 'true',
+  alignItems = 'center',
+  display = 'inline-flex',
+  element = 'SIBLING_BOX',
+  flexShrink = 0,
+  justifyContent = 'center',
+  transition = 'all 120ms',
+  ...props
```
Comment (gloriliale) [inline packages/paste-core/primitives/sibling-box/src/index.tsx:90]: o0o0o cleaner and more understandable to have these defaults up here. Hooray!

**Review by gloriliale — APPROVED** [review]
Lookin nice and customizable 😎

File: packages/paste-core/primitives/sibling-box/__tests__/siblingBox.test.tsx [hunk 2]
```diff
@@ -60,4 +65,43 @@ describe('SiblingBox render', () => {
       .toJSON();
     expect(tree).toMatchSnapshot();
   });
+
+  describe('HTML attributes', () => {
+    it('should set a default element data attribute', () => {
```
Comment (nkrantz) [inline packages/paste-core/primitives/sibling-box/__tests__/siblingBox.test.tsx:70]: Why did you add the `: void` return type on the next 2 `it` blocks but not this one?

**Review by nkrantz — APPROVED** [review]
Nice 👍

**Review by TheSisb — APPROVED** [review]

File: packages/paste-core/primitives/sibling-box/__tests__/siblingBox.test.tsx (hunk 2 above)
Comment (SiTaggart) [reply packages/paste-core/primitives/sibling-box/__tests__/siblingBox.test.tsx:70]: Copy paste... :hide:
